redis-7.2.yaml

```diff
@@ -146,3 +146,23 @@ update:
   enabled: true
   github:
     identifier: redis/redis
+
+test:
+  environment:
+    contents:
+      packages:
+        - busybox
+        - redis-cli
+  pipeline:
+    - runs: |
+        cat <<EOF >> /tmp/redis.conf
+        dbfilename dump.rdb
+        pidfile /tmp/redis_6379.pid
+        dir /tmp/
+        EOF
+
+        redis-server /tmp/redis.conf &
+        sleep 2 # wait for redist to start
+        redis-cli SET bike:1 "Process 134" || exit 1
+        redis-cli GET bike:1 | grep 'Process 134' || exit 1
+        kill `cat /tmp/redis_6379.pid`
```

Comment:
```suggestion
```

can we just ignore the cleanup? or does `melange test` run the process on the host?